examples/testNG-single-module/src/test/java/Test1.java

```diff
@@ -0,0 +1,24 @@
+import org.testng.annotations.Test;
+
+import static org.testng.AssertJUnit.assertTrue;
+
+public class Test1 {
```

Comment:
Similar tests allow to compare results between projects. Can you prepare tests similar to other tests in other example projects (like spock-single-module)? 
- Test1
  - test1, sleep 200ms
  - test2, ignored, sleep 200ms
  - test3, sleep 200ms
- Test2
  - test1, sleep 300ms
  - test2, ignored, sleep 300ms
  - test3, sleep 300ms, parametrized (2 data sets)
- Isolated test
  - test1, sleep 100ms
  - test2 sleep 100ms
  - test3, sleep 100ms

Moreover Test1 and Test2 should be run in parallel. 
This can be done by:
`build.gradle`:
```gradle
test {
    useTestNG() {
        suites 'src/test/resources/testng-parallel.xml', 'src/test/resources/testng-isolated.xml'
    }
}
```
`testng-isolated.xml`:
```xml
<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE suite SYSTEM "https://testng.org/testng-1.0.dtd" >
<suite name="Isolated" verbose="1">
    <test name="Isolated">
        <groups>
            <run>
                <include name="isolated"/>
            </run>
        </groups>
        <packages>
            <package name=".*"/>
        </packages>
    </test>
</suite>
```
`testng-parallel.xml`:
```xml
<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE suite SYSTEM "https://testng.org/testng-1.0.dtd" >
<suite name="Parallel" verbose="1" parallel="methods" thread-count="4">
    <test name="Parallel">
        <groups>
            <run>
                <include name="parallel"/>
            </run>
        </groups>
        <packages>
            <package name=".*"/>
        </packages>
    </test>
</suite>
```
and:
```
@Test(groups = {"parallel"})
public class Test2 {
```

```
@Test(groups = {"isolated"})
public class TestIsolated {
```

Reply:
Fixed

Reply:
Thanks. Two things are missing. Can you ignore Test2.test2 and parametrize Test2.test3?

Reply:
Fixed
I'm sorry for overlooking